README.md

```diff
@@ -144,6 +149,45 @@ Now that your signaling channel is created and the connected master is streaming
 
 Choose Start viewer to start live video streaming of the sample H264/Opus frames.
 
+## Setup IoT
+* To use IoT certificate to authenticate to KVS signaling, refer to [Controlling Access to Kinesis Video Streams Resources Using AWS IoT](https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/how-iot.html) for provisioning details.
+* A sample IAM policy for the IoT role looks like below, policy can be modified based on your permission requirement.
+
+```
+{
+   "Version":"2012-10-17",
+   "Statement":[
+      {
+          "Effect":"Allow",
+          "Action":[
+            "kinesisvideo:DescribeSignalingChannel",
+            "kinesisvideo:CreateSignalingChannel",
+            "kinesisvideo:GetSignalingChannelEndpoint",
+            "kinesisvideo:GetIceServerConfig",
+            "kinesisvideo:ConnectAsMaster",
+            "kinesisvideo:ConnectAsViewer"
```

Comment:
I don't think it's worth mentioning that in order to execute the integ tests you might need DeleteChannel too

Reply:
Do we expect them to run the integ test? The reason of starting adding this part I kind of consider it as an entry of how customer can use this to run examples and for their custom use case they might even need to remove/add some other permissions. Actually I wonder even ConnectAsViewer might not need to be here. Thoughts?

Reply:
I think we should just have the master permission only and have a NOTE section below about the viewer and delete in order to run integ tests.